modules/standard/OS.chpl

```diff
@@ -1210,42 +1214,54 @@ module OS {
   }
 
   /*
-     :class:`IOError` is the subclass of :class:`SystemError` that serves as the
+     :class:`IoError` is the subclass of :class:`SystemError` that serves as the
      base class for all errors regarding inputs and their formatting.
      They are typically not directly generated by the OS, but they are
      used and emitted by the IO module.
   */
-  class IOError : SystemError {
+  class IoError : SystemError {
     proc init(err: syserr, details: string = "") {
       super.init(err, details);
     }
   }
 
+  pragma "no doc"
+  deprecated "'IOError' is deprecated, please use 'IoError' instead"
+  class IOError {}
```

Comment:
Shouldn't this also subclass SystemError so that calls to methods on it in user code don't just break? Ditto all the other ones.

Reply:
Thanks! I thought I did that, but apparently not. I updated them now.